addons/im_livechat/static/src/core/web/livechat_channel_tag_edit.js

```diff
@@ -0,0 +1,88 @@
+import { Component, onWillStart, useEffect, useRef, useState } from "@odoo/owl";
+
+import { useService } from "@web/core/utils/hooks";
+import { useSequential } from "@mail/utils/common/hooks";
+import { useDebounced } from "@web/core/utils/timing";
+
+export class ChannelTagEdit extends Component {
+    static components = {};
+    static props = ["autofocus?", "thread?", "close?", "selectedTags?", "state?"];
```

Comment:
`thread` doesn't look optional, you need to guard `onClickSave` button or make it required.
`state` props don't look used.